SignalServiceKit/src/Network/API/TSNetworkManager.m

```diff
@@ -71,15 +69,6 @@ - (void)makeRequest:(TSRequest *)request
     OWSAssert(failureBlock);
 
     DDLogInfo(@"%@ Making request: %@", self.logTag, request);
-    if (!CurrentAppContext().isMainApp) {
-        if (![request isKindOfClass:[TSRecipientPrekeyRequest class]]
-            && ![request isKindOfClass:[TSSubmitMessageRequest class]]
-            && ![request isKindOfClass:[TSContactsIntersectionRequest class]]
-            && ![request isKindOfClass:[TSAllocAttachmentRequest class]]) {
-            // The SAE should only make requests directly related to message sending.
-            OWSFail(@"%@ Making unexpected request: %@", self.logTag, request);
```

Comment:
If we really wanted to keep doing this, we could add a `isAppExtensionCompatible` property or something like that.